tests/gentropy/dataset/test_study_locus_overlaps.py

```diff
@@ -118,10 +131,38 @@ def test_overlapping_peaks(
         [
             t.StructField("leftStudyLocusId", t.StringType()),
             t.StructField("rightStudyLocusId", t.StringType()),
+            t.StructField("rightStudyType", t.StringType()),
             t.StructField("chromosome", t.StringType()),
         ]
     )
     observed_df = spark.createDataFrame(observed, mock_schema)
     result_df = StudyLocus._overlapping_peaks(observed_df, intrastudy)
```

Comment:
The overlapping peak method now returns the right study type as well, so in the outer join later, both the right and left pile of data will have the `rightStudyType` column. Unfortunately this also means I had to touch this test.